core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -31,6 +31,7 @@ buildFarm:
       - .*integr8ly-integreatly-operator-release-v2.4-presubmits.yaml$
       - .*integr8ly-integreatly-operator-release-v2.6-presubmits.yaml$
       - .*integr8ly-integreatly-operator-release-v2.7-presubmits.yaml$
+      - .*kata-containers-kata-containers-main-presubmits.yaml$
```

Comment:
is there any particular reason why your jobs need to run in `build01` cluster?

Reply:
@droslean If I don't set this configuration then the job is scheduled on the api.ci cluster (ocp 3.x IUUC) and the build of one image breaks. I talked with @petr-muller about that problem, and one possible solution was to explicitly require the  `build01` cluster.

Reply:
@droslean any further concern on this or can I get it approved?